signer-service/src/constants/constants.js

```diff
@@ -15,6 +15,7 @@ require('dotenv').config();
 const PENDULUM_FUNDING_SEED = process.env.PENDULUM_FUNDING_SEED;
 const FUNDING_SECRET = process.env.FUNDING_SECRET;
 const MOONBEAM_EXECUTOR_PRIVATE_KEY = process.env.MOONBEAM_EXECUTOR_PRIVATE_KEY;
+const SEP10_MASTER_SECRET = process.env.SEP10_MASTER_SECRET;
```

Comment:
Let's maybe set this to the same value so that it's easy to change back if we decide to have two separate accounts in the future?
```suggestion
const SEP10_MASTER_SECRET = FUNDING_SECRET;
```